website/static/js/koHelpers.js

```diff
@@ -59,6 +59,31 @@ addExtender('trimmed', function(value) {
     return $.trim(value);
 });
 
+addExtender('trimmedhttp', function(value) {
+ //   console.log("trimmedhttp value is " + value);
+    value = $.trim(value);
+    if (!value || value.search(/^https?:\/\//i) === 0) {
+        return value;
+    }
+    return 'http://' + value;
+});
+
+addExtender('trimmedWebArray', function(values) {
+    if (values) {
+    for (var i = 0; i < values.length; i++) {   
+        console.log("trimmedhttp value is " + values[i] + "and values.length is " + values.length);
+        var value = values[i];
+        var value = $.trim(value);
+        if (!value || value.search(/^https?:\/\//i) === 0) {
```

Comment:
There's duplication here with the `trimmedhttp` extender. This can be refactored.